class.jetpack-connection-banner.php

```diff
@@ -48,20 +81,15 @@ function maybe_initialize_hooks( $current_screen ) {
 			return;
 		}
 
-		$random_connection_banner = Jetpack_Options::get_option( 'connection_banner_ab' );
-		if ( ! $random_connection_banner ) {
-			$random_connection_banner = mt_rand( 1, 2 );
-			Jetpack_Options::update_option( 'connection_banner_ab', $random_connection_banner );
-		}
-
-		if ( 1 == $random_connection_banner ) {
-			add_action( 'admin_notices', array( $this, 'render_legacy_banner' ) );
-		} else {
+		if ( self::check_ab_test_allowed() && 2 == self::get_random_connection_banner_value() ) {
```

Comment:
Just want to verify that when this expires, we want to discontinue the new banner completely?